Nathan-Chevalier/levelup-server
# Fixtures added for each table

# Fixtures generated for each table, properly seeding database
## Steps to Test
- Pull down the `fixtures` branch associated with this PR
- Open your terminal and navigate to the `levelup-server`
- Ensure that you are in a virtual environment
- Run the following commands separately
```
python3 manage.py loaddata game_types
python3 manage.py loaddata users
python3 manage.py loaddata tokens
python3 manage.py loaddata games
python3 manage.py loaddata events
```
- Verify seeding the database worked by checking `/db.sqlite3`
